knowledge/software/products/Maximo/attribution.txt

```diff
@@ -0,0 +1,4 @@
+Title of work: Maximo application Authorization
+Link to work: https://www.ibm.com/docs/en/mas-cd/maximo-manage/continuous-delivery?topic=groups-application-templates
```

Comment:
We can not accept things that aren't in our approved list for the upstream model.